src/app/content/components/Page/highlightManager.ts

```diff
@@ -76,10 +79,29 @@ const onSelectHighlight = (
   services.setPendingHighlight(highlight);
 });
 
-const createHighlighter = (services: Omit<HighlightManagerServices, 'highlighter'>) => {
+const createHighlighter = (services: Omit<HighlightManagerServices, 'highlighter'>, intl: IntlShape) => {
 
   const highlighter: Highlighter = new Highlighter(services.container, {
-    onClick: (highlight) => onClickHighlight({ ...services, highlighter }, highlight),
+    formatMessage: intl.formatMessage,
+    onClick: (highlight) => onFocusHighlight({ ...services, highlighter }, highlight),
+    onFocusIn: (highlight) => onFocusHighlight({ ...services, highlighter }, highlight),
```

Comment:
Btw. doesn't click automatically fire `focusin` event?

Reply:
theoretically in normal circumstances yes, but the way its implemented in highlighter the elements you click on and the element you tab focus aren't the same, so in this case you need both.

@PiotrKozlowski it might be a good idea for the future to simplify the interface to get rid of `onClick` and just call `onFocusIn` from the click handler instead